algo-dev-demo/scikit-learn-demo/README.md

```diff
@@ -1,67 +1,93 @@
 
-- Click on **“Add Collection”** under the “My Collections” section on your data collections page. Let's name ours "scikit_learn_demo"
+### Create the Algorithm
 
-- After you create your collection you can set the read and write access on your data collection. We are going to select **"Private"** since only you will be calling your algorithm in this instance. 
+From the home page click the **Create new** button at the top right of the navbar and select **Algorithm**.
+Let's go through the form together to create our algorithm.
 
-- Now, let's put some data into your newly created data collection. You can either drag and drop the files [scikit-demo-boston-regression.pkl](https://github.com/algorithmiaio/sample-apps/raw/master/algo-dev-demo/scikit-learn-demo/data/scikit-demo-boston-regression.pkl) and [boston_test_data.csv](https://github.com/algorithmiaio/sample-apps/raw/master/algo-dev-demo/scikit-learn-demo/data/boston_test_data.csv) or you can click **"Drop files here to upload"** from where you stored the repo on your computer.
+- Name the Algorithm `scikit_learn_demo`
+- On the **Source Code** section select Algorithmia and **Source Code visibility**: **Restricted**
+- On the **Environment** section select **Language**: **Python 3** and **Environment**: **Python 3.7**.
+- You can leave the other values as the default ones
+- Click on **Create new algorithm**
 
-## Create Your Algorithm
+### Add the source code
 
-Now we are ready to deploy our model.
+Now click on the **Source Code** tab to go to the Web IDE.
 
-### First create an algorithm
-- Click the **"Plus"** icon at the top right of the navbar
-- Let's go through the form together to create our algorithm
-- Click on the tab **"Source"** and you'll notice boilerplate code for Hello World.
-- Let's delete that code, and copy and paste the code from the file [boston-housing-prices.py](https://github.com/algorithmiaio/sample-apps/blob/master/algo-dev-demo/scikit-learn-demo/demo/boston-housing-prices.py)
-- Note that you'll need to change the name of the data collection to the one we created earlier. 
+- Open the file with the same name as the algorithm, on this case it would be: `scikit_learn_demo.py`
+- Delete that code, and copy and paste the code from the file [src/boston-housing-prices.py](https://github.com/algorithmiaio/sample-apps/blob/master/algo-dev-demo/scikit-learn-demo/src/boston-housing-prices.py)
+- Change the name of the data collection to the one we created earlier.
 
-Recall our data collection is called "scikit_learn_demo" and you'll need
-to change "YOUR_USERNAME" to your own username: `file_path =
-'data://YOUR_USERNAME/scikit_learn_demo/scikit-demo-boston-regression.pkl'`
+Recall our data collection is called `scikit_learn_demo` and you'll also need
+to change `USERNAME` to your own username on this line:
```

Comment:
Suggest changing "username" to "username or organization".

Reply:
done